src/pkg/track/track.go

```diff
@@ -11,6 +12,10 @@ import (
 	"github.com/spf13/pflag"
 )
 
+// maxMessagePerProperty is the maximum number of messages to include in a single tracking property.
+// 3 seems to be a reasonable number to avoid exceeding size limits.
```

Comment:
Can we updated this comment to describe how `3` was determined to be a reasonable number? If I want to change this later, what should I consider?